src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -59,19 +60,24 @@ interface IOptionSettlementEngine {
     error ExpiredOption(uint256 optionId, uint40 expiry);
 
     /// @notice This option cannot yet be exercised.
-    error ExerciseTooEarly();
+    /// @param optionId Supplied option ID.
+    error ExerciseTooEarly(uint256 optionId);
 
     /// @notice This option has no claims written against it.
-    error NoClaims();
+    /// @param optionId Supplied option ID.
+    error NoClaims(uint256 optionId);
 
     /// @notice This account has no claims.
-    error BalanceTooLow();
+    /// @param claimId Supplied claim ID.
+    error BalanceTooLow(uint256 claimId);
```

Comment:
I understand this error is likely to align on the ERC1155 impl, but the message seems slightly opaque. We could consider renaming this error as e.g. "**`RedeemerDoesNotOwnClaimId`**", and perhaps emit the owner of the claim ID supplied.